tutorials/Tutorial11_Pipelines.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>"It consists of two main sections: you define all objects (e.g. a reader) in <code>components</code>  and then stick them together to a pipeline in <code>pipelines</code>  . You can also re-use one component in multiple nodes of a pipeline or even across multiple pipelines. It will just be loaded once in memory and therefore doesn't hurt your resources more than actually needed. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>